django_cache_manager/cache_manager.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-class CacheManager(models.Manager, CacheInvalidateMixin):
+class CacheManager(CacheInvalidateMixin, models.Manager):
```

Comment:
yeah this is fine, although in this case it doesn't matter since the mixins are adding functionality that doesn't exist in the base